LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
why do we need to add this license file?

Reply:
@johnliu25 based on the net-istio.yaml from https://github.com/knative-extensions/net-istio/releases 
```
# Copyright 2019 The Knative Authors
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     https://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```
We need to attach the apache license in order to use the yaml from the knative opensource from https://github.com/knative-extensions/net-istio/releases 

I refer to https://github.com/nutanix-cloud-native/prism-go-client/blob/main/LICENSE